bintest/example.cpp

```diff
@@ -0,0 +1,41 @@
+#include <cstddef>
+class MyClass {
```

Comment:
It seems this could reuse the `bindgen-integration` test that is set up already, can't it?

I think that'd be preferable.

Reply:
I did not know that bindgen-integrat existed. Fixed in the newest commit.